### PR TITLE
Remove projects without ideas list...

### PR DESCRIPTION
**Pull request by ocefpaf**
... and add suggested hours for student application.

@leewujung and @rsignell-usgs based on your ideas page I believe your projects will be 350 hours, right? Google now offers two otions 175 or 350 hours.

@7yl4r I'm assuming, due to the amount of work in pyobis, we will need 350.

I'm keeping the erddapy one flexible and I'll let the student decide.

@MathewBiddle I also removed ioos_qc and compliance-checker b/c we never heard back from devs on those projects.

### CODE REVIEW

**Comment by leewujung** [comment]
@ocefpaf : yes I think 350 hours is appropriate. Thanks!

**Comment by 7yl4r** [comment]
The pyobis ideas can easily eat up 350hrs but we can target a subset of small milestones if there is some reason 175hrs would be better.

**Comment by ocefpaf** [comment]
> The pyobis ideas can easily eat up 350hrs but we can target a subset of small milestones if there is some reason 175hrs would be better.

Let's make it flexible then and expand our chances to get a nice proposal.

**Comment by ocefpaf** [comment]
@MathewBiddle we got feedback from all the active mentors. Do you mind taking a look at this PR and merge it if it is OK?

File: 2022/ideas-list.md [hunk 1]
```diff
@@ -2,11 +2,9 @@
...
-| ioos_qc | | https://github.com/ioos/ioos_qc |
-| IOOS compliance checker | | https://github.com/ioos/compliance-checker |
-| kerchunk | Efficient access to IOOS data in the cloud with netCDF, Zarr, and JSON using [kerchunk](https://fsspec.github.io/kerchunk/).  | https://github.com/ioos/gsoc/issues/14 |
-| echoshader | Enhance the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the data exploration and discovery process. Expanding on [echopype](https://github.com/OSOceanAcoustics/echopype) | https://github.com/ioos/gsoc/issues/16 |
+|**Project** |**Description**|**links**| **hours** |
+|------------|---------------|---------|-----------|
+| erddapy    | Separate erddapy into more functional layers, roughly following the [SQLAlchemy core/ORM](https://docs.sqlalchemy.org/en/14/intro.html#overview) model.                                                                          | https://github.com/ioos/erddapy/issues/228 | 175  or 350  |
```
Comment (MathewBiddle) [inline 2022/ideas-list.md:7]: Could you link erddapy in the description? https://github.com/ioos/erddapy

**Review by MathewBiddle — APPROVED** [review]
see my one comment, ignore if you want. Otherwise this looks good

**Comment by rsignell-usgs** [comment]
Yes, for our proposal we could do some good work for 175, but probably 3x good work for 350, because of the spin-up time!

File: 2022/ideas-list.md (hunk 1 above)
Comment (ocefpaf) [reply 2022/ideas-list.md]: Done! I also removed the SQLAlchemy doc link b/c that is not necessary here.

**Pull request merged**